### ISSUE TITLE
Can Katago more tough when playing handicaps

### ISSUE THREAD

**Issue by ceremony08** [lightvector/KataGo#520]
just like crafty move or forcing killing, the pros often use it

**Comment by fuhaoda** [comment]
What if you increase `staticScoreUtilityFactor`  and also increase `dynamicScoreUtilityFactor` in the configuration file? Will it be more aggressive to maximizing the scores?

**Comment by ceremony08** [comment]
> What if you increase `staticScoreUtilityFactor` and also increase `dynamicScoreUtilityFactor` in the configuration file? Will it be more aggressive to maximizing the scores?

YES, I have tested these configs,but it is not clever enough. when pros playing, they can make the situation chaotic, and Force the opponent to make mistakes. I think the PDA is not good enough at this point

**Comment by ceremony08** [comment]
We can make 2 or some moves with red color, which are looking for the chance to kill  by force， just like some pros, such as weixing tang.
